code/game/objects/items/storage/book.dm

```diff
@@ -141,7 +141,7 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 			smack = 0
 		else if(iscarbon(M))
 			var/mob/living/carbon/C = M
-			if(!istype(C.head, /obj/item/clothing/head/helmet))
+			if(!istype(C.head, (/obj/item/clothing/head/helmet || /obj/item/clothing/head/hardhat)))
```

Comment:
??

Reply:
i should have specified in pr but basically since i made the headbutt brain damage not work on hardhat and helmet guys thought i should do the same for bibles